With dev setup https://github.com/lucascolley/pixi-dev-scipystack/tree/smoke-docs, I cannot reproduce the failure:

```
scipy on 🎋 main [$!] is 📦 v1.16.0.dev0 via 🐍 
❯ pixi r smoke-docs 
✨ Pixi task (smoke-docs in doc): python dev.py smoke-docs
<snip>
================================================================================================= FAILURES =================================================================================================
_____________________________________________________________________________________ [doctest] scipy.interpolate.AAA ______________________________________________________________________________________
390     ...     warnings.simplefilter('ignore', RuntimeWarning)
391     ...     r = AAA(z, f(z), rtol=0, max_terms=50, clean_up=False)
392     >>> mask = np.abs(r.residues()) < 1e-13
393     >>> fig, axs = plt.subplots(ncols=2)
394     >>> axs[0].plot(r.poles().real[~mask], r.poles().imag[~mask], '.')
395     >>> axs[0].plot(r.poles().real[mask], r.poles().imag[mask], 'r.')
396 
397     Now we call the `clean_up` method to remove Froissart doublets.
398 
399     >>> with warnings.catch_warnings():
Expected:
    4
Got:
    3

/Users/lucascolley/programming/pixi-dev-scipystack/scipy/scipy/build-install/lib/python3.12/site-packages/scipy/interpolate/_bary_rational.py:399: DocTestFailure
____________________________________________________________________________________ [doctest] scipy.signal.place_poles ____________________________________________________________________________________
2822     array([[ 0.52072442+0.j, -0.08409372+0.j, -0.56847937+0.j,  0.74823657+0.j],
2823            [-0.04977751+0.j, -0.80872954+0.j,  0.13566234+0.j, -0.29322906+0.j],
2824            [-0.82266932+0.j, -0.19168026+0.j, -0.56348322+0.j, -0.43815060+0.j],
2825            [ 0.22267347+0.j,  0.54967577+0.j, -0.58387806+0.j, -0.40271926+0.j]])
2826 
2827     The absolute value of the determinant of X is a good indicator to check the
2828     robustness of the results, both ``'KNV0'`` and ``'YT'`` aim at maximizing
2829     it.  Below a comparison of the robustness of the results above:
2830 
2831     >>> abs(np.linalg.det(fsf1.X)) < abs(np.linalg.det(fsf2.X))
UNEXPECTED EXCEPTION: RuntimeWarning('divide by zero encountered in det')
Traceback (most recent call last):
  File "/Users/lucascolley/programming/pixi-dev-scipystack/scipy/.pixi/envs/doc/lib/python3.12/doctest.py", line 1368, in __run
    exec(compile(example.source, filename, "single",
  File "<doctest scipy.signal.place_poles[12]>", line 1, in <module>
  File "/Users/lucascolley/programming/pixi-dev-scipystack/scipy/.pixi/envs/doc/lib/python3.12/site-packages/numpy/linalg/_linalg.py", line 2383, in det
    r = _umath_linalg.det(a, signature=signature)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeWarning: divide by zero encountered in det
/Users/lucascolley/programming/pixi-dev-scipystack/scipy/scipy/build-install/lib/python3.12/site-packages/scipy/signal/_ltisys.py:2831: UnexpectedException
========================================================================================= short test summary info ==========================================================================================
FAILED scipy/interpolate/__init__.py::scipy.interpolate.AAA
FAILED scipy/signal/__init__.py::scipy.signal.place_poles
```

env:

```
scipy on 🎋 main [$!] is 📦 v1.16.0.dev0 via 🐍 took 1m37s 
❯ pixi ls -e doc
Environment: doc
Package                        Version         Build                 Size       Kind   Source
accessible-pygments            0.0.5           pyhd8ed1ab_0          1.3 MiB    conda  accessible-pygments
alabaster                      0.7.16          pyhd8ed1ab_0          17.9 KiB   conda  alabaster
anyio                          4.7.0           pyhd8ed1ab_0          110.1 KiB  conda  anyio
appnope                        0.1.4           pyhd8ed1ab_1          9.8 KiB    conda  appnope
argon2-cffi                    23.1.0          pyhd8ed1ab_1          18.2 KiB   conda  argon2-cffi
argon2-cffi-bindings           21.2.0          py312h024a12e_5       32.1 KiB   conda  argon2-cffi-bindings
arrow                          1.3.0           pyhd8ed1ab_1          97.6 KiB   conda  arrow
asttokens                      3.0.0           pyhd8ed1ab_1          27.5 KiB   conda  asttokens
attrs                          24.2.0          pyh71513ae_1          55 KiB     conda  attrs
babel                          2.16.0          pyhd8ed1ab_1          6.2 MiB    conda  babel
beautifulsoup4                 4.12.3          pyha770c72_1          115.3 KiB  conda  beautifulsoup4
beniget                        0.4.2.post1     pyhd8ed1ab_1          21.2 KiB   conda  beniget
blas-devel                     3.9.0           25_osxarm64_openblas  15.5 KiB   conda  blas-devel
bleach                         6.2.0           pyhd8ed1ab_1          129.8 KiB  conda  bleach
brotli                         1.1.0           hd74edd7_2            19.1 KiB   conda  brotli
brotli-bin                     1.1.0           hd74edd7_2            16.4 KiB   conda  brotli-bin
brotli-python                  1.1.0           py312hde4cb15_2       331.4 KiB  conda  brotli-python
bzip2                          1.0.8           h99b78c6_7            120 KiB    conda  bzip2
c-compiler                     1.8.0           hf48404e_1            6.1 KiB    conda  c-compiler
ca-certificates                2024.8.30       hf0a4a13_0            154.8 KiB  conda  ca-certificates
cached-property                1.5.2           hd8ed1ab_1            4 KiB      conda  cached-property
cached_property                1.5.2           pyha770c72_1          10.8 KiB   conda  cached_property
ccache                         4.10.1          hbe278c5_0            500.8 KiB  conda  ccache
cctools                        1010.6          hf67d63f_2            20.6 KiB   conda  cctools
cctools_osx-arm64              1010.6          h623e0ac_2            1.1 MiB    conda  cctools_osx-arm64
certifi                        2024.8.30       pyhd8ed1ab_0          159.9 KiB  conda  certifi
cffi                           1.17.1          py312h0fad829_0       274.6 KiB  conda  cffi
charset-normalizer             3.4.0           pyhd8ed1ab_1          46.4 KiB   conda  charset-normalizer
clang                          17.0.6          default_h360f5da_7    23.5 KiB   conda  clang
clang-17                       17.0.6          default_h146c034_7    699.2 KiB  conda  clang-17
clang_impl_osx-arm64           17.0.6          he47c785_23           17.5 KiB   conda  clang_impl_osx-arm64
clang_osx-arm64                17.0.6          h07b0088_23           20.7 KiB   conda  clang_osx-arm64
clangxx                        17.0.6          default_h360f5da_7    23.6 KiB   conda  clangxx
clangxx_impl_osx-arm64         17.0.6          h50f59cd_23           17.6 KiB   conda  clangxx_impl_osx-arm64
clangxx_osx-arm64              17.0.6          h07b0088_23           19.1 KiB   conda  clangxx_osx-arm64
click                          8.1.7           unix_pyh707e725_1     82.5 KiB   conda  click
cloudpickle                    3.1.0           pyhd8ed1ab_1          25.3 KiB   conda  cloudpickle
colorama                       0.4.6           pyhd8ed1ab_1          26.4 KiB   conda  colorama
colorlog                       6.9.0           pyh707e725_1          15.2 KiB   conda  colorlog
comm                           0.2.2           pyhd8ed1ab_1          11.8 KiB   conda  comm
compiler-rt                    17.0.6          h856b3c1_2            92.7 KiB   conda  compiler-rt
compiler-rt_osx-arm64          17.0.6          h832e737_2            9.9 MiB    conda  compiler-rt_osx-arm64
compilers                      1.8.0           hce30654_1            6.9 KiB    conda  compilers
contourpy                      1.3.1           py312hb23fbb9_0       239.9 KiB  conda  contourpy
coverage                       7.6.9           py312h998013c_0       354.4 KiB  conda  coverage
cxx-compiler                   1.8.0           h18dbf2f_1            6.1 KiB    conda  cxx-compiler
cycler                         0.12.1          pyhd8ed1ab_1          13.1 KiB   conda  cycler
cython                         3.0.11          py312hde4cb15_2       3.2 MiB    conda  cython
debugpy                        1.8.10          py312hd8f9ff3_0       2.4 MiB    conda  debugpy
decorator                      5.1.1           pyhd8ed1ab_1          13.7 KiB   conda  decorator
defusedxml                     0.7.1           pyhd8ed1ab_0          23.5 KiB   conda  defusedxml
docutils                       0.21.2          pyhd8ed1ab_1          393.3 KiB  conda  docutils
doit                           0.36.0          pyhd8ed1ab_0          65.6 KiB   conda  doit
entrypoints                    0.4             pyhd8ed1ab_1          11 KiB     conda  entrypoints
exceptiongroup                 1.2.2           pyhd8ed1ab_1          20 KiB     conda  exceptiongroup
execnet                        2.1.1           pyhd8ed1ab_1          37.9 KiB   conda  execnet
executing                      2.1.0           pyhd8ed1ab_1          27.7 KiB   conda  executing
fonttools                      4.55.3          py312h998013c_0       2.6 MiB    conda  fonttools
fortran-compiler               1.8.0           hc3477c4_1            6.1 KiB    conda  fortran-compiler
fqdn                           1.5.1           pyhd8ed1ab_1          16.3 KiB   conda  fqdn
freetype                       2.12.1          hadb7bae_2            582.5 KiB  conda  freetype
gast                           0.5.5           pyhd8ed1ab_0          23.5 KiB   conda  gast
gfortran                       13.2.0          h1ca8e4b_1            31.2 KiB   conda  gfortran
gfortran_impl_osx-arm64        13.2.0          h252ada1_3            17.6 MiB   conda  gfortran_impl_osx-arm64
gfortran_osx-arm64             13.2.0          h57527a5_1            34.4 KiB   conda  gfortran_osx-arm64
gmp                            6.3.0           h7bae524_2            356.6 KiB  conda  gmp
gmpy2                          2.1.5           py312h524cf62_3       144.5 KiB  conda  gmpy2
greenlet                       3.1.1           py312hde4cb15_0       227.5 KiB  conda  greenlet
h2                             4.1.0           pyhd8ed1ab_1          50.8 KiB   conda  h2
hpack                          4.0.0           pyhd8ed1ab_1          28.7 KiB   conda  hpack
hyperframe                     6.0.1           pyhd8ed1ab_1          16.8 KiB   conda  hyperframe
hypothesis                     6.122.3         pyha770c72_0          331.5 KiB  conda  hypothesis
icu                            75.1            hfee45f7_0            11.3 MiB   conda  icu
idna                           3.10            pyhd8ed1ab_1          48.6 KiB   conda  idna
imagesize                      1.4.1           pyhd8ed1ab_0          9.9 KiB    conda  imagesize
importlib-metadata             8.5.0           pyha770c72_1          28 KiB     conda  importlib-metadata
importlib_resources            6.4.5           pyhd8ed1ab_1          31.9 KiB   conda  importlib_resources
iniconfig                      2.0.0           pyhd8ed1ab_1          11.2 KiB   conda  iniconfig
intersphinx-registry           0.2412.7        pyhd8ed1ab_0          13.4 KiB   conda  intersphinx-registry
ipykernel                      6.29.5          pyh57ce528_0          116.8 KiB  conda  ipykernel
ipython                        8.30.0          pyh707e725_0          586.2 KiB  conda  ipython
isl                            0.26            imath32_h347afa1_101  800.7 KiB  conda  isl
isoduration                    20.11.0         pyhd8ed1ab_1          19.4 KiB   conda  isoduration
jedi                           0.19.2          pyhd8ed1ab_1          823.9 KiB  conda  jedi
jinja2                         3.1.4           pyhd8ed1ab_1          108.4 KiB  conda  jinja2
json5                          0.10.0          pyhd8ed1ab_1          30.8 KiB   conda  json5
jsonpointer                    3.0.0           py312h81bd7bf_1       17.4 KiB   conda  jsonpointer
jsonschema                     4.23.0          pyhd8ed1ab_1          72.5 KiB   conda  jsonschema
jsonschema-specifications      2024.10.1       pyhd8ed1ab_1          15.8 KiB   conda  jsonschema-specifications
jsonschema-with-format-nongpl  4.23.0          hd8ed1ab_1            7 KiB      conda  jsonschema-with-format-nongpl
jupyter-cache                  1.0.1           pyhff2d567_0          30.5 KiB   conda  jupyter-cache
jupyter_client                 8.6.3           pyhd8ed1ab_1          103.8 KiB  conda  jupyter_client
jupyter_core                   5.7.2           pyh31011fe_1          56.3 KiB   conda  jupyter_core
jupyter_events                 0.10.0          pyhd8ed1ab_1          20.9 KiB   conda  jupyter_events
jupyter_server                 2.14.2          pyhd8ed1ab_1          316.7 KiB  conda  jupyter_server
jupyter_server_terminals       0.5.3           pyhd8ed1ab_1          19.2 KiB   conda  jupyter_server_terminals
jupyterlab_pygments            0.3.0           pyhd8ed1ab_2          18.3 KiB   conda  jupyterlab_pygments
jupyterlab_server              2.27.3          pyhd8ed1ab_1          48.3 KiB   conda  jupyterlab_server
jupyterlite-core               0.4.5           pyhd8ed1ab_0          14 MiB     conda  jupyterlite-core
jupyterlite-pyodide-kernel     0.4.5           pyh885dcc9_0          234.4 KiB  conda  jupyterlite-pyodide-kernel
jupyterlite-sphinx             0.16.5          pyhd8ed1ab_0          21.3 KiB   conda  jupyterlite-sphinx
jupytext                       1.16.4          pyh80e38bb_0          102.1 KiB  conda  jupytext
kiwisolver                     1.4.7           py312h6142ec9_0       59.5 KiB   conda  kiwisolver
krb5                           1.21.3          h237132a_0            1.1 MiB    conda  krb5
lcms2                          2.16            ha0e7c42_0            207 KiB    conda  lcms2
ld64                           951.9           h39a299f_2            18.1 KiB   conda  ld64
ld64_osx-arm64                 951.9           h3f9b568_2            993.9 KiB  conda  ld64_osx-arm64
lerc                           4.0.0           h9a09cb3_0            210.7 KiB  conda  lerc
libblas                        3.9.0           25_osxarm64_openblas  15.5 KiB   conda  libblas
libbrotlicommon                1.1.0           hd74edd7_2            66.8 KiB   conda  libbrotlicommon
libbrotlidec                   1.1.0           hd74edd7_2            27.7 KiB   conda  libbrotlidec
libbrotlienc                   1.1.0           hd74edd7_2            273.1 KiB  conda  libbrotlienc
libcblas                       3.9.0           25_osxarm64_openblas  15.5 KiB   conda  libcblas
libclang-cpp17                 17.0.6          default_h146c034_7    11.8 MiB   conda  libclang-cpp17
libcxx                         19.1.5          ha82da77_0            507.6 KiB  conda  libcxx
libcxx-devel                   17.0.6          h86353a2_6            801.6 KiB  conda  libcxx-devel
libdeflate                     1.22            hd74edd7_0            52.8 KiB   conda  libdeflate
libedit                        3.1.20191231    hc8eb9b7_2            94.3 KiB   conda  libedit
libexpat                       2.6.4           h286801f_0            63.2 KiB   conda  libexpat
libffi                         3.4.2           h3422bc3_5            38.1 KiB   conda  libffi
libgfortran                    5.0.0           13_2_0_hd922786_3     107.6 KiB  conda  libgfortran
libgfortran-devel_osx-arm64    13.2.0          h5d7a38c_3            1.9 MiB    conda  libgfortran-devel_osx-arm64
libgfortran5                   13.2.0          hf226fd6_3            974 KiB    conda  libgfortran5
libglib                        2.82.2          h07bd6cf_0            3.5 MiB    conda  libglib
libhiredis                     1.0.2           hbec66e7_0            50.7 KiB   conda  libhiredis
libiconv                       1.17            h0d3ecfb_2            660.6 KiB  conda  libiconv
libintl                        0.22.5          h8414b35_3            79.3 KiB   conda  libintl
libjpeg-turbo                  3.0.0           hb547adb_1            534.7 KiB  conda  libjpeg-turbo
liblapack                      3.9.0           25_osxarm64_openblas  15.5 KiB   conda  liblapack
liblapacke                     3.9.0           25_osxarm64_openblas  15.5 KiB   conda  liblapacke
libllvm17                      17.0.6          h5090b49_2            23.5 MiB   conda  libllvm17
liblzma                        5.6.3           h39f12f2_1            96.8 KiB   conda  liblzma
libopenblas                    0.3.28          openmp_hf332438_1     4 MiB      conda  libopenblas
libpng                         1.6.44          hc14010f_0            257.2 KiB  conda  libpng
libsodium                      1.0.20          h99b78c6_0            161.1 KiB  conda  libsodium
libsqlite                      3.47.2          h3f77e49_0            830.6 KiB  conda  libsqlite
libtiff                        4.7.0           ha962b0a_2            361.7 KiB  conda  libtiff
libwebp-base                   1.4.0           h93a5062_0            281 KiB    conda  libwebp-base
libxcb                         1.17.0          hdb1d25a_0            316.1 KiB  conda  libxcb
libxml2                        2.13.5          h178c5d8_1            569.2 KiB  conda  libxml2
libzlib                        1.3.1           h8359307_2            45.3 KiB   conda  libzlib
llvm-openmp                    19.1.5          hdb05f8b_0            274.5 KiB  conda  llvm-openmp
llvm-tools                     17.0.6          h5090b49_2            20.9 MiB   conda  llvm-tools
markdown-it-py                 3.0.0           pyhd8ed1ab_1          62.9 KiB   conda  markdown-it-py
markupsafe                     3.0.2           py312h998013c_1       23.5 KiB   conda  markupsafe
matplotlib                     3.9.3           py312h1f38498_0       16.7 KiB   conda  matplotlib
matplotlib-base                3.9.3           py312hdbc7e53_0       7.4 MiB    conda  matplotlib-base
matplotlib-inline              0.1.7           pyhd8ed1ab_1          14.1 KiB   conda  matplotlib-inline
mdit-py-plugins                0.4.2           pyhd8ed1ab_1          41.2 KiB   conda  mdit-py-plugins
mdurl                          0.1.2           pyhd8ed1ab_1          14.1 KiB   conda  mdurl
meson                          1.6.0           pyhd8ed1ab_1          641.5 KiB  conda  meson
meson-python                   0.17.1          pyh70fd9c4_1          72.5 KiB   conda  meson-python
mistune                        3.0.2           pyhd8ed1ab_1          64.4 KiB   conda  mistune
mpc                            1.3.1           h8f1351a_1            102.3 KiB  conda  mpc
mpfr                           4.2.1           hb693164_3            337.4 KiB  conda  mpfr
mpmath                         1.3.0           pyhd8ed1ab_1          429.4 KiB  conda  mpmath
munkres                        1.1.4           pyh9f0ad1d_0          12.2 KiB   conda  munkres
myst-nb                        1.1.2           pyhd8ed1ab_0          61.4 KiB   conda  myst-nb
myst-parser                    4.0.0           pyhd8ed1ab_0          71.1 KiB   conda  myst-parser
nbclient                       0.10.1          pyhd8ed1ab_0          27.2 KiB   conda  nbclient
nbconvert-core                 7.16.4          pyhff2d567_2          184.1 KiB  conda  nbconvert-core
nbformat                       5.10.4          pyhd8ed1ab_1          98.6 KiB   conda  nbformat
ncurses                        6.5             h7bae524_1            783.5 KiB  conda  ncurses
nest-asyncio                   1.6.0           pyhd8ed1ab_1          11.3 KiB   conda  nest-asyncio
ninja                          1.12.1          h420ef59_0            109.9 KiB  conda  ninja
numpy                          2.2.0           py312h41c6370_0       6.2 MiB    conda  numpy
numpydoc                       1.8.0           pyhd8ed1ab_1          56.7 KiB   conda  numpydoc
openblas                       0.3.28          openmp_hea878ba_1     2.9 MiB    conda  openblas
openjpeg                       2.5.3           h8a3d83b_0            311.9 KiB  conda  openjpeg
openssl                        3.4.0           h39f12f2_0            2.8 MiB    conda  openssl
overrides                      7.7.0           pyhd8ed1ab_0          29.5 KiB   conda  overrides
packaging                      24.2            pyhd8ed1ab_2          58.8 KiB   conda  packaging
pandocfilters                  1.5.0           pyhd8ed1ab_0          11.4 KiB   conda  pandocfilters
parso                          0.8.4           pyhd8ed1ab_1          73.5 KiB   conda  parso
pcre2                          10.44           h297a79d_2            604.5 KiB  conda  pcre2
pexpect                        4.9.0           pyhd8ed1ab_1          52.3 KiB   conda  pexpect
pickleshare                    0.7.5           pyhd8ed1ab_1004       11.5 KiB   conda  pickleshare
pillow                         11.0.0          py312haf37ca6_0       39.8 MiB   conda  pillow
pip                            24.3.1          pyh8b19718_0          1.2 MiB    conda  pip
pkg-config                     0.29.2          hde07d2e_1009         48.6 KiB   conda  pkg-config
pkginfo                        1.12.0          pyhd8ed1ab_1          29.5 KiB   conda  pkginfo
pkgutil-resolve-name           1.3.10          pyhd8ed1ab_2          10.4 KiB   conda  pkgutil-resolve-name
platformdirs                   4.3.6           pyhd8ed1ab_1          20 KiB     conda  platformdirs
pluggy                         1.5.0           pyhd8ed1ab_1          23 KiB     conda  pluggy
ply                            3.11            pyhd8ed1ab_3          47.9 KiB   conda  ply
pooch                          1.8.2           pyhd8ed1ab_1          52.8 KiB   conda  pooch
prometheus_client              0.21.1          pyhd8ed1ab_0          47.9 KiB   conda  prometheus_client
prompt-toolkit                 3.0.48          pyha770c72_1          263.5 KiB  conda  prompt-toolkit
psutil                         6.1.0           py312h0bf5046_0       481.9 KiB  conda  psutil
pthread-stubs                  0.4             hd74edd7_1002         8.2 KiB    conda  pthread-stubs
ptyprocess                     0.7.0           pyhd8ed1ab_1          19 KiB     conda  ptyprocess
pure_eval                      0.2.3           pyhd8ed1ab_1          16.3 KiB   conda  pure_eval
pybind11                       2.13.6          pyh1ec8472_2          182 KiB    conda  pybind11
pybind11-global                2.13.6          pyh415d2e4_2          174.9 KiB  conda  pybind11-global
pycparser                      2.22            pyh29332c3_1          107.5 KiB  conda  pycparser
pydata-sphinx-theme            0.16.0          pyhd8ed1ab_0          1.5 MiB    conda  pydata-sphinx-theme
pydevtool                      0.3.0           pyhd8ed1ab_0          14.7 KiB   conda  pydevtool
pygments                       2.18.0          pyhd8ed1ab_1          856.2 KiB  conda  pygments
pyobjc-core                    10.3.2          py312hb9d441b_0       473.2 KiB  conda  pyobjc-core
pyobjc-framework-cocoa         10.3.2          py312hb9d441b_0       371.5 KiB  conda  pyobjc-framework-cocoa
pyparsing                      3.2.0           pyhd8ed1ab_2          90.2 KiB   conda  pyparsing
pyproject-metadata             0.9.0           pyhd8ed1ab_1          20.9 KiB   conda  pyproject-metadata
pyproject_hooks                1.2.0           pyhd8ed1ab_1          15.2 KiB   conda  pyproject_hooks
pysocks                        1.7.1           pyha55dd90_7          20.6 KiB   conda  pysocks
pytest                         8.3.4           pyhd8ed1ab_1          253.1 KiB  conda  pytest
pytest-cov                     6.0.0           pyhd8ed1ab_1          25.6 KiB   conda  pytest-cov
pytest-timeout                 2.3.1           pyhd8ed1ab_2          18.9 KiB   conda  pytest-timeout
pytest-xdist                   3.6.1           pyhd8ed1ab_1          37.3 KiB   conda  pytest-xdist
python                         3.12.8          hc22306f_1_cpython    12.4 MiB   conda  python
python-build                   1.2.2.post1     pyhff2d567_1          24.5 KiB   conda  python-build
python-dateutil                2.9.0.post0     pyhff2d567_1          217.3 KiB  conda  python-dateutil
python-fastjsonschema          2.21.1          pyhd8ed1ab_0          221 KiB    conda  python-fastjsonschema
python-json-logger             2.0.7           pyhd8ed1ab_0          13.1 KiB   conda  python-json-logger
python_abi                     3.12            5_cp312               6.1 KiB    conda  python_abi
pythran                        0.17.0          pyh27800b7_1          1.9 MiB    conda  pythran
pytz                           2024.2          pyhd8ed1ab_1          181.5 KiB  conda  pytz
pyyaml                         6.0.2           py312h024a12e_1       182.8 KiB  conda  pyyaml
pyzmq                          26.2.0          py312hf8a1cbd_3       353.2 KiB  conda  pyzmq
qhull                          2020.2          h420ef59_5            504.3 KiB  conda  qhull
readline                       8.2             h92ec313_1            244.5 KiB  conda  readline
referencing                    0.35.1          pyhd8ed1ab_1          41.2 KiB   conda  referencing
requests                       2.32.3          pyhd8ed1ab_1          57.3 KiB   conda  requests
rfc3339-validator              0.1.4           pyhd8ed1ab_1          10 KiB     conda  rfc3339-validator
rfc3986-validator              0.1.1           pyh9f0ad1d_0          7.6 KiB    conda  rfc3986-validator
rich                           13.9.4          pyhd8ed1ab_1          181.3 KiB  conda  rich
rich-click                     1.8.3           pyhd8ed1ab_0          33 KiB     conda  rich-click
rpds-py                        0.22.3          py312hcd83bfe_0       311.4 KiB  conda  rpds-py
scipy-doctest                  1.6             pyhd8ed1ab_0          55.6 KiB   conda  scipy-doctest
send2trash                     1.8.3           pyh31c8845_1          22.6 KiB   conda  send2trash
setuptools                     75.6.0          pyhff2d567_1          756.1 KiB  conda  setuptools
sigtool                        0.1.3           h44b9a77_0            205.3 KiB  conda  sigtool
six                            1.17.0          pyhd8ed1ab_0          16 KiB     conda  six
sniffio                        1.3.1           pyhd8ed1ab_1          14.7 KiB   conda  sniffio
snowballstemmer                2.2.0           pyhd8ed1ab_0          57.4 KiB   conda  snowballstemmer
sortedcontainers               2.4.0           pyhd8ed1ab_0          25.7 KiB   conda  sortedcontainers
soupsieve                      2.5             pyhd8ed1ab_1          35.9 KiB   conda  soupsieve
sphinx                         7.4.7           pyhd8ed1ab_0          1.3 MiB    conda  sphinx
sphinx-copybutton              0.5.2           pyhd8ed1ab_0          17.4 KiB   conda  sphinx-copybutton
sphinx-design                  0.6.1           pyhd8ed1ab_1          894.8 KiB  conda  sphinx-design
sphinxcontrib-applehelp        2.0.0           pyhd8ed1ab_1          29.1 KiB   conda  sphinxcontrib-applehelp
sphinxcontrib-devhelp          2.0.0           pyhd8ed1ab_1          24 KiB     conda  sphinxcontrib-devhelp
sphinxcontrib-htmlhelp         2.1.0           pyhd8ed1ab_1          32.1 KiB   conda  sphinxcontrib-htmlhelp
sphinxcontrib-jsmath           1.0.1           pyhd8ed1ab_1          10.2 KiB   conda  sphinxcontrib-jsmath
sphinxcontrib-qthelp           2.0.0           pyhd8ed1ab_1          26.3 KiB   conda  sphinxcontrib-qthelp
sphinxcontrib-serializinghtml  1.1.10          pyhd8ed1ab_1          28 KiB     conda  sphinxcontrib-serializinghtml
sqlalchemy                     2.0.36          py312h0bf5046_0       3.3 MiB    conda  sqlalchemy
stack_data                     0.6.3           pyhd8ed1ab_1          26.4 KiB   conda  stack_data
tabulate                       0.9.0           pyhd8ed1ab_2          36.7 KiB   conda  tabulate
tapi                           1300.6.5        h03f4b80_0            202.8 KiB  conda  tapi
terminado                      0.18.1          pyh31c8845_0          22.2 KiB   conda  terminado
threadpoolctl                  3.5.0           pyhc1e730c_0          23 KiB     conda  threadpoolctl
tinycss2                       1.4.0           pyhd8ed1ab_0          27.6 KiB   conda  tinycss2
tk                             8.6.13          h5083fa2_1            3 MiB      conda  tk
toml                           0.10.2          pyhd8ed1ab_1          21.6 KiB   conda  toml
tomli                          2.2.1           pyhd8ed1ab_1          18.7 KiB   conda  tomli
tornado                        6.4.2           py312hea69d52_0       822.8 KiB  conda  tornado
traitlets                      5.14.3          pyhd8ed1ab_1          107.5 KiB  conda  traitlets
types-python-dateutil          2.9.0.20241206  pyhd8ed1ab_0          21.6 KiB   conda  types-python-dateutil
typing-extensions              4.12.2          hd8ed1ab_1            9.8 KiB    conda  typing-extensions
typing_extensions              4.12.2          pyha770c72_1          38.7 KiB   conda  typing_extensions
typing_utils                   0.1.0           pyhd8ed1ab_1          14.8 KiB   conda  typing_utils
tzdata                         2024b           hc8b5060_0            119.5 KiB  conda  tzdata
unicodedata2                   15.1.0          py312h0bf5046_1       363.8 KiB  conda  unicodedata2
uri-template                   1.3.0           pyhd8ed1ab_1          23.4 KiB   conda  uri-template
urllib3                        2.2.3           pyhd8ed1ab_1          95.8 KiB   conda  urllib3
wcwidth                        0.2.13          pyhd8ed1ab_1          31.8 KiB   conda  wcwidth
webcolors                      24.11.1         pyhd8ed1ab_0          18 KiB     conda  webcolors
webencodings                   0.5.1           pyhd8ed1ab_3          15.1 KiB   conda  webencodings
websocket-client               1.8.0           pyhd8ed1ab_1          45.6 KiB   conda  websocket-client
wheel                          0.45.1          pyhd8ed1ab_1          61.5 KiB   conda  wheel
xorg-libxau                    1.0.11          hd74edd7_1            13.2 KiB   conda  xorg-libxau
xorg-libxdmcp                  1.1.5           hd74edd7_0            18.1 KiB   conda  xorg-libxdmcp
yaml                           0.2.5           h3422bc3_2            86 KiB     conda  yaml
zeromq                         4.3.5           hc1bb282_7            275 KiB    conda  zeromq
zipp                           3.21.0          pyhd8ed1ab_1          21.3 KiB   conda  zipp
zlib                           1.3.1           h8359307_2            75.8 KiB   conda  zlib
zstandard                      0.23.0          py312h15fbf35_1       323 KiB    conda  zstandard
zstd                           1.5.6           hb46c0d2_0            395.6 KiB  conda  zstd
```

--- a/scipy/sparse/__init__.py
+++ b/scipy/sparse/__init__.py
@@ -253,7 +253,7 @@ is the same:
 Now we can compute norm of the error with:
 
 >>> err = norm(x-x_)
->>> err < 1e-10
+>>> err < 1e-9
 True
 
 It should be small :)

--- a/scipy/sparse/linalg/_norm.py
+++ b/scipy/sparse/linalg/_norm.py
@@ -144,7 +144,7 @@ def norm(x, ord=None, axis=None):
             raise ValueError('Duplicate axes given.')
         if ord == 2:
             # Only solver="lobpcg" supports all numpy dtypes
-            _, s, _ = svds(x, k=1, solver="lobpcg", maxiter=50)
+            _, s, _ = svds(x, k=1, solver="lobpcg")
             return s[0]
         elif ord == -2:
             raise NotImplementedError

